corehq/apps/prototype/static/prototype/js/example/knockout_pagination_webpack.main.js

```diff
@@ -0,0 +1,70 @@
+// base level imports need to figure out how to always include these
+
+//need to figure out gettext mapping
+import 'hqwebapp/js/bootstrap5/common';
+import 'hqwebapp/js/bootstrap5/base_main';
+
+import ko from 'knockout';
+import mapping from 'ko.mapping';
+ko.mapping = mapping;
+
+
+import $ from 'jquery';
```

Comment:
Musing over previous conversations we've had about eventually moving off of jQuery, do you think we could draw a bright line saying, "If you're working in react you don't use jQuery"?

Reply:
100%! i am in favor of this